File: src/main/java/be4rjp/sclat/manager/BungeeCordMgr.java

package be4rjp.sclat.manager;

import be4rjp.sclat.Sclat;
import com.google.common.io.ByteArrayDataOutput;
import com.google.common.io.ByteStreams;
import org.bukkit.entity.Player;

/**
 *
 * @author Be4rJP
 */
public class BungeeCordMgr {
	public static void PlayerSendServer(Player player, String servername) {
		ByteArrayDataOutput out = ByteStreams.newDataOutput();
		out.writeUTF("Connect");
		out.writeUTF(servername);
		player.sendPluginMessage(Sclat.getPlugin(), "BungeeCord", out.toByteArray());
	}
}

File: src/main/java/be4rjp/sclat/manager/BungeeCordMgr.kt
package be4rjp.sclat.manager

import be4rjp.sclat.plugin
import com.google.common.io.ByteStreams
import org.bukkit.entity.Player

/**
 *
 * @author Be4rJP
 */
object BungeeCordMgr {
    @JvmStatic
    fun PlayerSendServer(player: Player, servername: String) {
        val out = ByteStreams.newDataOutput()
        out.writeUTF("Connect")
        out.writeUTF(servername)
        player.sendPluginMessage(plugin, "BungeeCord", out.toByteArray())
    }
}
